src/helpers/metrics-playground-opener.js

```diff
@@ -0,0 +1,3 @@
+export const metricsPlaygroundOpener = () => {
+  window.open('https://azion.com', '_blank')
```

Comment:
Acredito que esta URl seja diferente , pois está levando não para o grphQl playground mas sim para o site da azion.

Reply:
Dummy code. Quando a url for ajustada o teste vai quebrar.